source/_reference-a-f.html.md.erb

```diff
@@ -5024,7 +5024,1007 @@ Lists all cross-sheet references for the sheet.
+
+### SelectionRange Object
+
+     |     |     |
+-----|-----|-----|
+**sourceColumnId1** | number | Defines beginning edge of range when specifying one or more columns.
+**sourceColumnId2** | number | Defines ending edge of range when specifying one or more columns.
+**sourceRowId1** | number | Defines beginning edge of range when specifying one or more rows.
+**sourceRowId2** | number | Defines ending edge of range when specifying one or more rows.
+
+### ShortcutWidgetContent Object
+
+     |     |     |
+-----|-----|-----|
+**shortcutData** | [ShortcutDataItem](#shortcutdataitem-object)[] | Array of ShortcutDataItem objects
+
+### ShortcutDataItem Object
+
+     |     |     |
+-----|-----|-----|
+**attachmentType** | string | Attachment type (one of **BOX_COM**, **DROPBOX**, **EGNYTE**, **EVERNOTE**, **FILE**, **GOOGLE_DRIVE**, **LINK**, **ONEDRIVE**, or **SMARTSHEET**)
+**hyperlink** | [Hyperlink](#hyperlink-object) | Hyperlink object
+**label** | string | Label for the data point
+**labelFormat** | string | [formatDescriptor](#formatting)
+**mimeType** | string | MIME type if available for attachment type
+**order** | integer | The display order for the ShortcutWidgetItem object
+
+### TitleWidgetContent Object
+
+     |     |     |
+-----|-----|-----|
+**backgroundColor** | hex | The hex color, for instance #E6F5FE
+**htmlContent** | string | HTML snippet to render title
+
+### WebcontentWidgetContent Object
+
+     |     |     |
+-----|-----|-----|
+**url** | string | The URL
+
+### WidgetHyperlink Object
+
+     |     |     |
+-----|-----|-----|
+**interactionType** | string | Specifies what happens when a user clicks the widget. Valid values: <ul><li>**DISTI_COPY** -- a distribution link, only available if org has distribution links enabled</li><li>**NONE** -- do nothing, no other attribute required</li><li>**SMARTSHEET_ITEM** -- open a Smartsheet item that is not the SOURCE_SHEET, must also have one of the following:</li><ul><li>**folderId**</li><li>**reportId**</li><li>**sheetId**</li><li>**sightId**</li><li>**workspaceId**</li></ul><li>**SOURCE_SHEET** -- open the container where the widget data comes from, no other attribute required</li><li>**WEB** -- launch the URL, must also have **url** attribute</li></ul>
+**folderId** | number | If **interactionType** = **SMARTSHEET_ITEM** this is the folder to open
+**reportId** | number | If **interactionType** = **SMARTSHEET_ITEM** this is the report to open
+**sheetId** | number | If **interactionType** = **SMARTSHEET_ITEM** this is the sheet to open
+**sightId** | number | If **interactionType** = **SMARTSHEET_ITEM** this is the dashboard to open
```

Comment:
_This_ `sightId` is the only "real" `sightId` property that's returned from the API. At a glance, it appears that every other iteration of `sightId` is actually just `id`, and could be documented by `dashboardId` instead. But, I think that kinda throws a wrench into the clarity of the whole document, when this _one_ property is called `sightId`. Totally up to you on what you think is clearest. 

:+1: from me, as is or changed

Reply:
Agree. It's too confusing to change it.